### DISCUSSION TITLE
公衆WIFI  WI2_freeを使用した瞬間

### DISCUSSION THREAD

**Discussion by pazu0000** [community/community#146558] [New to GitHub] [answered]
gtthubデスクトップを開くと
<img width="596" alt="スクリーンショット 2024-12-08 16 40 29" src="https://github.com/user-attachments/assets/68f12eff-4ac3-4e60-8a56-bbff942c7efa">

ファイファイルのエラーが出ました。このような経験のある方はいらっしゃいませんでしょうか？
アドバイス、どうかよろしくお願いします。

**Comment by tuves** [comment, marked as answer]
Hi @pazu0000, thanks for participating!

Unfortunately, we don’t currently have moderators for languages other than English. Until that changes, we need to ask that everyone use English here in the GitHub Community when posting to ensure discussions are adhering to our [Code of Conduct](https://github.com/community/community/blob/920a928ffe3ae4863b64a4ea26917046e11f5c1a/CODE_OF_CONDUCT.md). 

We’ll be locking this post for now, but please feel encouraged to create a new discussion in English if you still need support.

**Comment by peme969** [comment]
Yeah and i think the user is using a japanese server when their location is different which may cause that. 
*i feel like this user is from another country (eg. north korea, china, russia) that doesnt support that (eg. mailing from another country’s server) because i have had emails from jp mail servers, though this is just a guess*
